README.md

```diff
@@ -13,43 +13,24 @@ This roles installs rsyslog v8 for the log shipping.
 For more information on rsyslog please visit [rsyslog docs](http://www.rsyslog.com/doc/v8-stable/).
 
 
-
-
-## ansible.cfg
-
-This role is designed to work with merge "hash_behaviour". Make sure your
-ansible.cfg contains these settings
-
-```INI
-[defaults]
-hash_behaviour = merge
-```
-
-
-
-
 ## Installation and Dependencies
```

Comment:
Can you remove the link to 'ansible.cfg' at the top of the README, might as well add link to the Builtin log configs section whilst you're at it.

Reply:
Thank you